helpers/__init__.py

```diff
@@ -0,0 +1,3 @@
+import logging
```

Comment:
you should be able to add a single ___nit___py at the project root level

Reply:
Tried without success